Sources/Safetensors/Safetensors.swift

```diff
@@ -231,34 +51,26 @@ public enum Safetensors {
         let data = try Data(contentsOf: url, options: .mappedIfSafe)
         return try decode(data)
     }
-
+    
     ///  Decode Data object to ParsedSafetensors object.
     /// - Parameter data: `Data` object containing the encoded data
     /// - Returns: `ParsedSafetensors` object containing the decoded data
     public static func decode(_ data: Data) throws -> ParsedSafetensors {
-        guard data.count >= 8 else {
+        guard data.count >= MemoryLayout<Int>.size else {
```

Comment:
docs says that it's explicitly 8 bytes https://github.com/huggingface/safetensors?tab=readme-ov-file#format

Reply:
btw. I think it's redundant, you're already checking it inside `let result = try HeaderDecoder().decode(data)`